### DISCUSSION TITLE
WTA Changing Breeds Roll for Buttons like Werewolf has.

### DISCUSSION THREAD

**Discussion by mnwolfboy** [JohanFalt/Foundry_WoD20#676] [Q&A] [answered]
Would like to add that the changing breeds character sheets would benefit from having the same macro buttons like roll for shapechange, frenzy, and remain active like the werewolf sheet has.

**Comment by JohanFalt** [comment]
Can all shifters with rage do this?

**Comment by mnwolfboy** [comment]
Do they not all shapeshift with same mechanics? I thought it was Stamina and primal urge for changing breeds as well. There no roll for shapechange button, granted we can manually do it, but werewolf sheets have the button, changing breeds do not. I also thought it be same mechanics for the other two. I think frenzy might vary, but remain active I think would be the same. Im no expert on the mechanics though.

**Comment by JohanFalt** [comment, marked as answer]
Added issue.
